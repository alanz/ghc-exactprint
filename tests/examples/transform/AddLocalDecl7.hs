module AddLocalDecl7 where

d1 = 1
  where -- c1
        w1 = 1

d2 = 1
  where w2 = 1

d3 = 1
  where

d4 = 1

d5 = 1
  where -- c5

d6 = 1 where

d7 = 1 where -- c7
