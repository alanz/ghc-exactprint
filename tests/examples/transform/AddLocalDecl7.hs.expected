module AddLocalDecl7 where

d1 = 1
  where nn = 2
        -- c1
        w1 = 1

d2 = 1
  where nn = 2
        w2 = 1

d3 = 1
  where
  nn = 2

d4 = 1
  where
    nn = 2

d5 = 1
  where
  nn = 2 -- c5

d6 = 1 where
  nn = 2

d7 = 1 where
  nn = 2 -- c7
